extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {loadPromise} from '../../../src/event-helper';
+
+class AmpBridPlayer extends AMP.BaseElement {
+
+  /** @override */
+  createdCallback() {
+    this.preconnect.url('https://services.brid.tv');
+    this.preconnect.url('https://cdn.brid.tv');
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return isLayoutSizeDefined(layout);
+  }
+
+  /** @override */
+  layoutCallback() {
+    const width = this.element.getAttribute('width');
+    const height = this.element.getAttribute('height');
+
+    const partnerID = AMP.assert(
+        this.element.getAttribute('data-partner'),
+        'The data-partner attribute is required for <amp-brid-player> %s',
+        this.element);
+    const playerID =  AMP.assert(this.element.getAttribute('data-player'),
+        'The data-player attribute is required for <amp-brid-player> %s',
+        this.element);
+    const feedID = AMP.assert(
+        (this.element.getAttribute('data-video') ||
+        this.element.getAttribute('data-playlist')),
+        'Either the data-video or the data-playlist attributes must be specified for <amp-brid-player> %s',
+        this.element);
+
+    let feedType;
+
+    if(this.element.getAttribute('data-video')){
+      feedType = "video";
+    } else if(this.element.getAttribute('data-playlist')){
+      feedType = "playlist";
+    }
+
+    //Create iframe
+    const iframe = document.createElement('iframe');
+    let src = "https://services.brid.tv/services/iframe/"+feedType+"/"+feedID+"/"+partnerID+"/"+playerID+"/0/1";
```

Comment:
Add `encodeURIComponent` around the stuff that is being interpolated into the URL